src/main/java/org/netcracker/learningcenter/utils/AnalysisStatus.java

```diff
@@ -0,0 +1,32 @@
+package org.netcracker.learningcenter.utils;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class AnalysisStatus {
```

Comment:
I think you do not need this class at all you can hold statuses in ReportService